pastis/matrix_generation/matrix_building_numerical.py

```diff
@@ -289,7 +289,7 @@ def calculate_unaberrated_contrast_and_normalization(instrument, design=None, re
         ppl.plot_direct_coro_dh(direct_psf, coro_psf, dh_mask, outpath)
 
     if return_coro_simulator:
-        return contrast_floor, norm, coro_simulator
+        return contrast_floor, norm, dh_mask, coro_simulator
```

Comment:
This change is not back-compatible so we will have to see how to deal with it. Let's talk offline about this.

Reply:
May be we can do the same change for all chronograph? It will be better for performances.

Reply:
It probably would, but testing all cases needs a lot of time. I think the easiest, although not the best, path forward is to revert this to what it was, and make a note to change it at a later point. In any case, such intrusive refactors are better done as the only action point in a separate PR, to make sure nothing goes wrong.

Reply:
@davidbourgeois I encourage you to make a new branch in which you keep these changes, and only these changes. Then we can use this branch to do the full refactor and test it.